home-application/pom.xml

```diff
@@ -57,6 +57,11 @@
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter-actuator</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.liquibase</groupId>
```

Comment:
Haven't we got it from home-data-migration already?

Reply:
I thought so as well but without this dependency application just doesn't work.

Reply:
I understand the problem. Due to the reason that we packed home-data-migration into fat jar we have the whole classpath from it. And when we import it in any place we duplicate the classpath. We can avoid it keeping original home-data-migration or simply remove problematic dependency from it. Please see: https://github.com/ita-social-projects/Home/pull/171/files